testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/AbstractHttpClientTest.java

```diff
@@ -195,6 +195,20 @@ void successfulRequestWithParent(String method) {
         });
   }
 
+  @Test
+  void successfulRequestWithNotSampledParent() throws InterruptedException {
+    String method = "GET";
+    URI uri = resolveAddress("/success");
+    int responseCode = testing.runWithNonRecordingSpan(() -> doRequest(method, uri));
+
+    assertThat(responseCode).isEqualTo(200);
+
+    // sleep to ensure no spans are emitted
+    Thread.sleep(1000);
+
+    assertThat(testing.traces()).isEmpty();
```

Comment:
How does this verify that the proper headers are being sent?  Would you expect the server to generate a trace if it wasn't?

Reply:
ya 👍